README.md

```diff
@@ -212,6 +213,10 @@ have to update this test to make it pass.
 ---------------------------------------------
 ```
 
+Another approach to testing callbacks is to whitelist existing callbacks, and
```

Comment:
I think I'd actually like to move this right up to the introduction. The examples are really great :+1: Could we even say "at GitHub we test callbacks like this"?